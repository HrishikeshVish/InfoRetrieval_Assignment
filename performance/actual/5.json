["https://archive.org/details/FOXNEWSW_20120412_030000_The_OReilly_Factor#start/2038/end/2073", "https://archive.org/details/BBCNEWS_20170106_150000_BBC_News#start/2496/end/2531", "https://archive.org/details/FOXNEWSW_20181204_080000_Fox_News__Night_With_Shannon_Bream#start/2772/end/2807", "https://archive.org/details/MSNBCW_20170603_120000_MSNBC_Live#start/3338/end/3373", "https://archive.org/details/FOXNEWSW_20120412_060000_The_Five#start/2035/end/2070", "https://archive.org/details/BBCNEWS_20170106_150000_BBC_News#start/2496/end/2531", "https://archive.org/details/BBCNEWS_20170106_150000_BBC_News#start/2496/end/2531", "https://archive.org/details/BBCNEWS_20200112_030000_BBC_News#start/1430/end/1465", "https://archive.org/details/BBCNEWS_20170106_150000_BBC_News#start/2496/end/2531", "https://archive.org/details/BBCNEWS_20170612_130000_BBC_News#start/2751/end/2786"]