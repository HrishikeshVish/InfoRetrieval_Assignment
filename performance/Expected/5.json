["https://archive.org/details/MSNBCW_20161026_000000_All_In_With_Chris_Hayes#start/2227/end/2262", "https://archive.org/details/FOXNEWSW_20120412_000000_The_OReilly_Factor#start/2035/end/2070", "https://archive.org/details/FOXNEWSW_20120412_060000_The_Five#start/2035/end/2070", "https://archive.org/details/MSNBCW_20170603_120000_MSNBC_Live#start/3338/end/3373", "https://archive.org/details/FOXNEWSW_20120412_030000_The_OReilly_Factor#start/2038/end/2073", "https://archive.org/details/MSNBCW_20170422_170000_MSNBC_Live#start/2189/end/2224", "https://archive.org/details/FOXNEWSW_20170603_100000_Fox_and_Friends_Saturday#start/404/end/439", "https://archive.org/details/BBCNEWS_20200112_030000_BBC_News#start/1430/end/1465", "https://archive.org/details/BBCNEWS_20170612_130000_BBC_News#start/2751/end/2786", "https://archive.org/details/FOXNEWSW_20181204_080000_Fox_News__Night_With_Shannon_Bream#start/2772/end/2807"]